health-services/project-factory/src/server/utils/microplanUtils.ts

```
@@ -0,0 +1,383 @@
+            if (cell.col < statusColIndex) {  // Unlock cells before the status column
+              cell.protection = { locked: false };
+            }
+          });
+        }
+      });
+    }
+  });
+}
+
+export function lockWithConfig(sheet: any) {
+  for (let row = 1; row <= parseInt(config.values.unfrozeTillRow); row++) {
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)